src/test/java/com/epam/test_generator/controllers/CaseControllerTest.java

```diff
@@ -387,20 +375,7 @@ public void testUpdateCase_return422whenUpdateCaseWithLessThanTheRequiredPriorit
             .andExpect(status().isBadRequest());
 
         verify(casesService, times(0))
-            .updateCase(eq(SIMPLE_SUIT_ID), eq(SIMPLE_CASE_ID), any(CaseDTO.class));
-    }
-
-    @Test
-    public void testUpdateCase_return422whenUpdateCaseWithNullDescription() throws Exception {
-        caseDTO.setDescription(null);
-
-        mockMvc.perform(put("/suits/" + SIMPLE_SUIT_ID + "/cases/" + SIMPLE_CASE_ID)
-            .contentType(MediaType.APPLICATION_JSON)
-            .content(mapper.writeValueAsString(caseDTO)))
-            .andExpect(status().isBadRequest());
-
-        verify(casesService, times(0))
-            .updateCase(eq(SIMPLE_SUIT_ID), eq(SIMPLE_CASE_ID), any(CaseDTO.class));
+            .updateCase(eq(SIMPLE_SUIT_ID), eq(SIMPLE_CASE_ID), any(EditCaseDTO.class));
```

Comment:
Since these 2 cases are not relevant anymore, I would suggest to write another test case that reflects changed business logic.